src/OpenFeature.Providers.GOFeatureFlag/OpenFeature.Providers.GOFeatureFlag.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>OpenFeature.Providers.GOFeatureFlag</PackageId>
+    <VersionNumber>0.2.1</VersionNumber> <!--x-release-please-version -->
+    <VersionPrefix>$(VersionNumber)</VersionPrefix>
+    <AssemblyVersion>$(VersionNumber)</AssemblyVersion>
+    <FileVersion>$(VersionNumber)</FileVersion>
+    <Description>GO Feature Flag provider for .NET</Description>
+    <Authors>Thomas Poignant</Authors>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <WasmVersion>v1.45.0</WasmVersion>
+    <OutputDirectory>$(ProjectDir)WasmModules</OutputDirectory>
+    <SourceFile>$(ProjectDir)wasm-releases/evaluation/gofeatureflag-evaluation_$(WasmVersion).wasi</SourceFile>
+    <DestinationFile>$(OutputDirectory)\gofeatureflag-evaluation.wasi</DestinationFile>
+    <RootNamespace>OpenFeature.Providers.GOFeatureFlag</RootNamespace>
+  </PropertyGroup>
+
+  <Target Name="PrepareWasmModule" BeforeTargets="CoreCompile">
+    <Warning Text="Source file '$(SourceFile)' does not exist. Skipping file move operation." Condition="!Exists($(SourceFile))"/>
+    <MakeDir Directories="$(OutputDirectory)" Condition="!Exists($(OutputDirectory))"/>
+    <Copy SourceFiles="$(SourceFile)" DestinationFiles="$(DestinationFile)" Condition="Exists($(SourceFile))"/>
+  </Target>
+
+  <!-- documentation -->
+  <ItemGroup>
+    <None Include="README.md" Pack="true" PackagePath="\"/>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="OpenFeature" Version="2.2.0"/>
```

Comment:
I would advise using a newer version so that later you can expand and add DI support.

Reply:
I'll move to 2.7.0